index.js

```diff
@@ -0,0 +1,2 @@
+import SvgRenderer from 'src';
+export default SvgRenderer;
```

Comment:
Do you still need an `index.js`?

Reply:
I was using it to test, but it seems we don't need it anymore.